rules_bot.py

```diff
@@ -160,7 +179,6 @@ def off_on_topic(bot, update, groups):
             'Come join us!',
             disable_web_page_preview=True, parse_mode=ParseMode.MARKDOWN)
 
-
```

Comment:
Please don't remove this like. We want 2 lines between top level functions.

Reply:
Uninterntional. Thanks!